Hippo.Web/Services/YamlService.cs

```diff
@@ -26,24 +27,32 @@ public async Task<string> Get(User currentUser, AccountCreateModel accountCreate
         {
             var sponsorAccount = await _dbContext.Accounts.Include(a => a.Owner).SingleAsync(a => a.Id == accountCreateModel.SponsorId);
 
-            var sb = new StringBuilder();
-            sb.AppendLine("sponsor:");
-            sb.AppendLine($"    accountname: {sponsorAccount.Name}");
-            sb.AppendLine($"    name: {sponsorAccount.Owner.Name}");
-            sb.AppendLine($"    email: {sponsorAccount.Owner.Email}");
-            sb.AppendLine($"    kerb: {sponsorAccount.Owner.Kerberos}");
-            sb.AppendLine($"    iam: {sponsorAccount.Owner.Iam}");
-            sb.AppendLine($"    mothra: {sponsorAccount.Owner.MothraId}");
-            sb.AppendLine();
-            sb.AppendLine("account:");
-            sb.AppendLine($"    name: {currentUser.Name}");
-            sb.AppendLine($"    email: {currentUser.Email}");
-            sb.AppendLine($"    kerb: {currentUser.Kerberos}");
-            sb.AppendLine($"    iam: {currentUser.Iam}");
-            sb.AppendLine($"    mothra: {currentUser.MothraId}");
-            sb.AppendLine($"    key: \"{accountCreateModel.SshKey}\"");
-
-            return sb.ToString();
+            var yaml =  new Serializer();
+            
+            return yaml.Serialize(
+                new
+                {
+                    sponsor = new
+                    {
+                        accountname = sponsorAccount.Name,
+                        name = sponsorAccount.Owner.Name,
+                        email = sponsorAccount.Owner.Email,
+                        kerb = sponsorAccount.Owner.Kerberos,
+                        iam = sponsorAccount.Owner.Iam,
+                        mothra = sponsorAccount.Owner.MothraId,
+                        cluster = sponsorAccount.Cluster
+                    },
+                    account = new
+                    {
+                        name = currentUser.Name,
+                        email = currentUser.Email,
+                        kerb = currentUser.Kerberos,
+                        iam = currentUser.Iam,
+                        mothra = currentUser.MothraId,
+                        key = accountCreateModel.SshKey

```

Comment:
Does it need the extra quotes?

Reply:
pardon? I'm not sure I understand

Reply:
The code that was deleted looked like this
sb.AppendLine($"    key: \"{accountCreateModel.SshKey}\"");

With escaped double quotes. I don't remember why we did that originally.

Reply:
Ah I see. You could be right, something else may expect the quotes. Thank you. Lemme check with Omen to be sure it's not a requirement on the puppet side. I'll see if I can do a more in-depth test on my local hippo. If anything I can just add the quotes back in the new version

Reply:
@jSylvestre see #189: you don't want to be in the business of formatting the YAML yourself; the serializer will handle escaping and quoting as-needed automagically.